src/tensors/cpu/tensor_operators.cpp

```diff
@@ -1071,15 +1065,55 @@ void LayerNormalization(Tensor out_,
 #pragma omp simd
     for(int i = 0; i < cols; ++i) {
       float t = alpha[alphaStride * i] * ((sp[i] - mean) / sigma);
-      if(beta != nullptr) {
+      if(hasBeta) {
         t += beta[betaStride * i];
```

Comment:
Consider removing the curlies here, to keep the code vertically more compact

Reply:
I tracked down where that brace came from.  It was @emjotde in d9d66f41 .  Then @frankseide touched the next line in 77a420740c79782d251bfe43a10279e3cfb10781 but left the brace as is.  Just following the style you set.

Reply:
3078927